.profile

```diff
@@ -69,8 +69,11 @@ source_if_exists "$HOME/.rbenv/completions/rbenv.bash"
 ##################################################
 # python
 ##################################################
+# WARNING: `pyenv init -` no longer sets PATH.
 export PYENV_ROOT="$HOME/.pyenv"
 export PATH="$PYENV_ROOT/bin:$PATH"
+eval "$(pyenv init --path)"
```

Comment:
It's not clear to me why `if command -v pyenv 1>/dev/null 2>&1; then eval "$(pyenv init - &)"; fi` would no longer suffice. Won't this line just fail if the `pyenv` command does not exist, also (vs aborting in line 78)?

Reply:
the issue is running `pyenv init --path` because `pyenv init -` seems to have stopped working with a recent pyenv update (see links in my updated PR). You're right this would fail if it doesn't exist though, so I updated the code to just run the --path version after the check